src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -490,16 +490,25 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 {
                     assert(!instrIsRMW);
 
-                    // Special handling for ConvertTo* APIs
-                    // Just need to change the opt here.
+                    insScalableOpts soptEmb = INS_SCALABLE_OPTS_NONE;
```

Comment:
I actually missed this and have changet locally, but basically, we should be doing something like:

```c++
insOpts embOpt = opt;
```

then set it appropriately below in `Extract*` or `ConvertTo*` case and then use it only for places where we will use the `insEmbMask` instruction.